docs/content/guides/dev/_index.md

```diff
@@ -4,19 +4,13 @@ weight: 60
 description: Extending Gloo Gateway's functionality with the use of plugins
 ---
 
-## Intro
-
+## Guides
 
-Gloo Gateway invites developers to extend Gloo Gateway's functionality and adapt to new use cases via the addition of plugins. 
+This Section includes the following developer guides:
 
-Gloo Gateway's plugin based architecture makes it easy to extend functionality in a variety of areas:
+{{% children description="true" %}}
 
-- [Gloo Gateway's API](https://github.com/solo-io/gloo/tree/main/projects/gloo/api/v1): extensible through the use of [Protocol Buffers](https://developers.google.com/protocol-buffers/) along with [Solo-Kit](https://github.com/solo-io/solo-kit)
-- [Service Discovery Plugins](https://github.com/solo-io/gloo/blob/main/projects/gloo/pkg/discovery/discovery.go#L21): automatically discover service endpoints from catalogs such as [Kubernetes](https://github.com/solo-io/gloo/tree/main/projects/gloo/pkg/plugins/kubernetes) and [Consul](https://github.com/solo-io/gloo/tree/main/projects/gloo/pkg/plugins/consul)
-- [Function Discovery Plugins](https://github.com/solo-io/gloo/blob/main/projects/discovery/pkg/fds/interface.go#L31): annotate services with information discovered by polling services directly (such as OpenAPI endpoints and gRPC methods).
-- [Routing Plugins](https://github.com/solo-io/gloo/blob/main/projects/gloo/pkg/plugins/plugin_interface.go#L53): customize what happens to requests when they match a route or virtual host
-- [Upstream Plugins](https://github.com/solo-io/gloo/tree/main/projects/gloo/pkg/plugins): customize what happens to requests when they are routed to a service
-- **Operators for Configuration**: Gloo Gateway exposes its intermediate language for proxy configuration via the {{< protobuf name="gloo.solo.io.Proxy" display="Proxy">}} Custom Resource, allowing operators to leverage Gloo Gateway for multiple use cases. With the optional [Gloo Gateway GraphQL module]({{< versioned_link_path fromRoot="/guides/graphql/" >}}), you can set up API gateway and GraphQL server functionality for your apps, without running in the same process (or even the same container) as Gloo Gateway.
+> Note: the Controller tutorial does not require modifying any Gloo Gateway code.
```

Comment:
what controller are we referring to here?

Reply:
The line existed prior to my changes and I felt it belonged near the list of guides. 

I assume it's referencing https://github.com/solo-io/gloo/blob/5352bc414c0b69d6158762cb0b656dad2e7cf16b/docs/content/guides/dev/example-proxy-controller.txt.